frontend/js/require.config.js

```diff
@@ -63,7 +69,7 @@ require.config({
 
         "mediaelementplayer": {
             exports: "mejs",
-            deps: ["jquery"]
+            deps: ["require"]
```

Comment:
Also, since we don't use the jQuery integration of ME.js anymore, my previous `deps` declaration here can go away, too (PR incoming). And adding one for `'require'` does not seem to be necessary as well.

Reply:
I understand, we don't need the reps anymore, I tested in Chrome and works without any deps.